File: CS/WinSample.Module/DomainObject1.cs
using System;
using System.ComponentModel;

using DevExpress.Xpo;
using DevExpress.Data.Filtering;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace WinSample.Module {
    [DefaultClassOptions]
    public class DomainObject1 : BaseObject {
        public DomainObject1(Session session) : base(session) { }
        private int _Property1;
        public int Property1 {
            get { return _Property1; }
            set { SetPropertyValue("Property1", ref _Property1, value); }
        }
        private int _Property2;
        public int Property2 {
            get { return _Property2; }
            set { SetPropertyValue("Property2", ref _Property2, value); }
        }
        private int _Property3;
        public int Property3 {
            get { return _Property3; }
            set { SetPropertyValue("Property3", ref _Property3, value); }
        }
        private int _Property4;
        public int Property4 {
            get { return _Property4; }
            set { SetPropertyValue("Property4", ref _Property4, value); }
        }
        private int _Property5;
        public int Property5 {
            get { return _Property5; }
            set { SetPropertyValue("Property5", ref _Property5, value); }
        }
        private int _Property6;
        public int Property6 {
            get { return _Property6; }
            set { SetPropertyValue("Property6", ref _Property6, value); }
        }
    }
}

File: VB/WinSample.Module/DomainObject1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace WinSample.Module
	<DefaultClassOptions> _
	Public Class DomainObject1
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private _Property1 As Integer
		Public Property Property1() As Integer
			Get
				Return _Property1
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property1", _Property1, value)
			End Set
		End Property
		Private _Property2 As Integer
		Public Property Property2() As Integer
			Get
				Return _Property2
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property2", _Property2, value)
			End Set
		End Property
		Private _Property3 As Integer
		Public Property Property3() As Integer
			Get
				Return _Property3
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property3", _Property3, value)
			End Set
		End Property
		Private _Property4 As Integer
		Public Property Property4() As Integer
			Get
				Return _Property4
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property4", _Property4, value)
			End Set
		End Property
		Private _Property5 As Integer
		Public Property Property5() As Integer
			Get
				Return _Property5
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property5", _Property5, value)
			End Set
		End Property
		Private _Property6 As Integer
		Public Property Property6() As Integer
			Get
				Return _Property6
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Property6", _Property6, value)
			End Set
		End Property
	End Class
End Namespace
